src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,56 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
-    @Override
+    private static final int MAX_SIZE = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_SIZE];
+        values = (V[]) new Object[MAX_SIZE];
+    }
+
     public void put(K key, V value) {
+        int index = indexOf(key);
+        if (index != -1) {
+            values[index] = value;
+        } else {
+            keys[size] = key;
+            values[size] = value;
+            size++;
+        }
     }
 
-    @Override
     public V get(K key) {
-        return null;
+        int index = indexOf(key);
+        if (index != -1) {
+            return values[index];
+        } else {
+            return null;
+        }
     }
 
-    @Override
     public int size() {
+        return size;
+    }
+
+    private int indexOf(K key) {
+        for (int i = 0; i < size; i++) {
+            if (keysEqual(keys[i], key)) {
```

Comment:
Why do you need separate method here? 
```suggestion
            if (key == keys[i] || key != null && key.equals(keys[i])) {
```